pkgs/by-name/ca/carto/package.nix

```
@@ -19,6 +19,11 @@ buildNpmPackage rec {
 
   dontNpmBuild = true;
 
+  postInstall = ''
+    # removing broken symlinks
+    find $out -xtype l -exec unlink {} \;
```

It's good practice to quote all bash variables.
```suggestion
    find "$out" -xtype l -exec unlink {} \;
```